openapi/backoffice/openapi.yaml

```diff
@@ -185,6 +203,9 @@ paths:
           $ref: '#/responses/NotFound'
         409:
           $ref: '#/responses/Conflict'
+      security:
+        - Bearer: [ ]
+          UserRole: [ ]
```

Comment:
Do we need to replicate this security section for each operation ?

Reply:
Created global security configuration